roles/ansible-keylime-tpm20/tasks/keylime.yml

```diff
@@ -15,6 +15,11 @@
     mode: 0644
     remote_src: yes
 
+- name: Convert keylime.conf to new config format
+  shell: "/usr/bin/python3 convert_config.py && mkdir /etc/keylime && cp agent.conf verifier.conf tenant.conf registrar.conf ca.conf logging.conf /etc/keylime/"
```

Comment:
I would pull the creation of `/etc/keylime` out into it's own task. Ansible is really good at making these kinds of common shell commands better and more repeatable (idempotent), so except for invoking custom commands (like `convert_config.py`) there's probably a better way to do it in an ansible task. See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#examples

Reply:
@mpeters Thanks for the pointer - I've also pulled out the `copy` command into its own task too.